docs/Configuration/yaml-files.md

```diff
@@ -86,7 +86,9 @@ Policies can be specified inline in your `default.yml`, `teams/team-name.yml`, o
 
 ### Options
 
-For possible options, see the parameters for the [Add policy API endpoint](https://fleetdm.com/docs/rest-api/rest-api#add-policy).
+For possible options, see the parameters for the [Edit team policy API endpoint](https://fleetdm.com/docs/rest-api/rest-api#edit-team-policy). 
```

Comment:
@rachaelshaw do you know why `software_title_id` and `script_id` are only available in the [edit team policy](https://fleetdm.com/docs/rest-api/rest-api#edit-team-policy) endpoint? And not the [edit policy (global) endpoint](https://fleetdm.com/docs/rest-api/rest-api#edit-policy).

<img width="695" height="251" alt="Screenshot 2025-07-22 at 10 25 07 AM" src="https://github.com/user-attachments/assets/8eab563b-11ef-40a2-9029-3e5030ca3db5" />

Reply:
@noahtalerman I'm assuming it's because you can only add those automations on team policies in the UI:
<img width="558" height="399" alt="Screenshot 2025-07-30 at 3 31 43 PM" src="https://github.com/user-attachments/assets/5c40a70c-888c-4ecb-9d6c-b07f9bac2303" />